# Question 1
-----

$$P(head) = 0.25$$

$$P(not heads) = 0.75$$

$$P(at least one head) = 1 - P(no heads)$$

$$ = 1 - (0.75)^5$$

$$ = 0.763$$

In [8]:
greater = 1 - pow(0.75, 14)
print(greater)

0.9821820519864559


The coin needs to be tossed 14 times for a probability of at least one head to be greater than 0.98

Alternative Solution:

$$1 - (0.75)^n \ge 0.98$$

$$0.75^n \le 0.02$$

$$n\log{0.75} \le \log{0.02}$$

$$n \ge \frac{\log{0.02}}{\log{0.75}}$$

In [10]:
import math
(math.log(0.02)) / (math.log(0.75))

13.598424719648978

A coin flip is discreet hence, you need at least 14 coin flips to obtain at least one head with a greater than 98% chance.